app/assets/stylesheets/application.scss

```diff
@@ -10,3 +10,27 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+body {
+  background-color: $background-gray;
+}
+
+h1 {
+  font-family: 'Koulen';
+  font-size: 4vw;
+}
+
+h2 {
+  font-family: 'Koulen';
+  font-size: 2vw;
+}
+
+h3 {
+  font-family: 'Koulen';
+  font-size: 1vw;
+}
```

Comment:
Since the font family is the same, we can select the h1, h2,h3 and assign font family

e.g 

h1,h2,h3 {
font-family: 'Koulen';
  font-size: 2vw;
  }